plugins/bbcode/plugin.js

```diff
@@ -157,6 +157,12 @@
 					// as "span" with an attribute marker.
 					if ( part == 'email' || part == 'img' )
 						attribs.bbcode = part;
+                                            
+                                        //special handling of left, right and center tags
+                                        if(part == 'left' || part == 'right' || part == 'center'){
+                                                styles[ stylesMap[ part ] ] = part;
```

Comment:
This code could be simplified and handled within the `optionPart` `if`, a couple lines above.